include/inexor/vulkan-renderer/vkresult_to_text.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
```

Comment:
This file is only used by the exception class, maybe place it into `vulkan_renderer/exception`.
Also put the definition into a cpp.

Reply:
I moved it into the new `VulkanException` class. (See pushed changes later..)